### ISSUE TITLE
Вырвиглазные тепловизоры/термалки

### ISSUE THREAD

**Issue by Motzord** [TauCetiStation/TauCetiClassic#6095]
#### Подробное описание проблемы
Палитра синдиочков/термалок хоса/ мезонок вырвиглазные и мешают игрокам использовать данные предметы
#### Что должно было произойти
ХОСУ/инженерам/нюке комфортно подрубать термалы/мезонки и играть с ними
#### Что произошло на самом деле
предметы неюзабельны из-за того что это неприятно игрокам.
#### Как повторить
Побегать в предметах описанных выше, осознать что глаза внезапно начали болеть
#### Дополнительная информация:
![image](https://user-images.githubusercontent.com/45365635/92995628-6ff21880-f505-11ea-860c-b4a5ed167bff.png)

**Comment by UsadoSmokk** [comment]
Согласен, однако....все идет в угоду баланса...иначе это будет найтвижен на минималках или что-то вроде этого.

**Comment by KIBORG04** [comment]
Баланс? Хехехе...

**Comment by LudwigVonChesterfield** [comment]
@UsadoSmokk балансировать предмет выдирая глаза игроку это какая-то идиотия.

**Comment by Chip11-n** [comment]
Это можно закрывать
